sys/net/routing/nhdp/nhdp.c

```diff
@@ -100,7 +101,10 @@ kernel_pid_t nhdp_start(void)
         /* Configure periodic timer message to refresh metric values */
         if (nhdp_pid != KERNEL_PID_UNDEF) {
             metric_interval = timex_from_uint64(DAT_REFRESH_INTERVAL * SEC_IN_USEC);
-            vtimer_set_msg(&metric_timer, metric_interval, nhdp_pid, NHDP_METRIC_TIMER, NULL);
+            metric_msg.type = NHDP_METRIC_TIMER;
```

Comment:
is my font displaying something wrong here, or is this and the subsequent lines really once off by one white-space?